src/sentry/workflow_engine/models/workflow_fire_history.py

```diff
@@ -0,0 +1,20 @@
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import CharField, FlexibleForeignKey, region_silo_model, sane_repr
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.uuid import UUIDField
+
+
+@region_silo_model
+class WorkflowFireHistory(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    workflow = FlexibleForeignKey("workflow_engine.Workflow")
+    group = FlexibleForeignKey("sentry.Group")
+    event_id = CharField(max_length=32)
+    notification_uuid = UUIDField(auto_add=True, unique=True)
```

Comment:
I don't really know how this col is used, but I just want to check that it's valid to always have a notification uuid here?

Reply:
currently no rows in RuleFireHistory have a null `notification_uuid` so it's ok to require